hive-event-listeners/apiary-gluesync-listener/src/main/java/com/expediagroup/apiary/extensions/gluesync/listener/metrics/MetricService.java

```diff
@@ -0,0 +1,25 @@
+package com.expediagroup.apiary.extensions.gluesync.listener.metrics;
+
+import java.util.Optional;
+
+import org.apache.hadoop.hive.common.metrics.common.Metrics;
+import org.apache.hadoop.hive.common.metrics.common.MetricsFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MetricService {
+
+  private static final Logger log = LoggerFactory.getLogger(MetricService.class);
+
+  public Optional<Long> incrementCounter(String name) {
+    try {
+      Metrics metrics = MetricsFactory.getInstance();
```

Comment:
I'd normally think we should do this in the constructor but I see we do it like this in other places, so maybe there is some catch?